books/sociology/bake

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '7.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :sociology) do |doc|
+  include Kitchen::Directions
+
+  # Set overrides
+  doc.selectors.override(
+    page_summary: 'section.section-summary',
+    reference: 'section.references'
+  )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  # Remove data-type="description" from body metadata placed in Sociology 2e
+  book.body.search('div[data-type="metadata"] > div[data-type="description"]').trash
```

Comment:
This could be a little slow as it is searching the entire book.  Is it possible for us to have a check to know which edition we are in

```ruby
edition = book.first('head').first('title').text.match(/(\d)e$/)[1].to_i
```

Then 

```ruby
if 2 == edition
  # remove the descriptions in metadata
end
```

Also I don't think you need `book.body.search` can just be `book.search`?

Reply:
@jpslav The above edition variable works. Thank you for the idea! About the other question wouldn't it be to general to use `book.search`instead of a body search and cause make the baking slower? I just need to remove metadata direct which are direct children of the body.

Reply:
Almost everything is included in the `body` element, so I don't think it is a speed thing, just makes the line a little shorter.